src/browser/preload_scripts.ts

```diff
@@ -81,7 +81,7 @@ function downloadScript(identity: Identity, preloadScript: PreloadScript): Promi
 export async function loadScripts(identity: Identity): Promise<PreloadScriptWithContent[]|any> {
     const options = Window.getOptions(identity);
     const { preloadScripts } = convertToElectron(options);
-    const promises = preloadScripts.map((preloadScript: PreloadScript) => loadScript(identity, preloadScript));
+    const promises = preloadScripts ? preloadScripts.map((preloadScript: PreloadScript) => loadScript(identity, preloadScript)) : [];
```

Comment:
I think the problem might lie earlier. Why doesn't this catch it https://github.com/HadoukenIO/core/blob/develop/src/browser/convert_options.js#L348

Reply:
IMO, we should either fix this in [here](https://github.com/HadoukenIO/core/blob/develop/src/browser/convert_options.js#L258) instead
--- or ---
do this: `const { preloadScripts = [] } = convertToElectron(options);`

Reply:
@whyn07m3 will make that change.